modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -715,13 +715,18 @@ public MediaPackage addTrack(InputStream in, String fileName, MediaPackageElemen
       job = serviceRegistry.updateJob(job);
       String elementId = UUID.randomUUID().toString();
       logger.info("Start adding track {} from input stream on mediapackage {}", elementId, mediaPackage);
+      if (fileName.length() > 75) {
```

Comment:
Maybe set this as a variable at the beginning of the class? final int MAX_FILENAME_LENGTH=75 ?

Reply:
Done